webapp/src/Controller/API/ClarificationController.php

```diff
@@ -27,7 +27,6 @@ class ClarificationController extends AbstractRestController
      * @param Request $request
      * @return \Symfony\Component\HttpFoundation\Response
      * @Rest\Get("")
-     * @Security("is_granted('ROLE_JURY') or is_granted('ROLE_JUDGEHOST') or is_granted('ROLE_API_READER')")
```

Comment:
I was first confused that we now allow anyone to view all clarifications, but then realized this is constrained in the QueryBuilder. Might be good to add a comment here.